build.gradle

```diff
@@ -14,31 +13,27 @@ group = 'com.zlad.gradle'
 version = resolveVersion()
 
 dependencies {
-    implementation "com.julianpeeters:avrohugger-core_$scalaMajorVersion:$avrohuggerVersion"
-    implementation "com.julianpeeters:avrohugger-filesorter_$scalaMajorVersion:$avrohuggerVersion"
+    implementation "com.julianpeeters:avrohugger-core_$scalaBinaryVersion:$avrohuggerVersion"
+    implementation "com.julianpeeters:avrohugger-filesorter_$scalaBinaryVersion:$avrohuggerVersion"
 
-    testImplementation("org.spockframework:spock-core:$spockVersion") {
-        exclude group: 'org.codehaus.groovy'
-    }
+    testImplementation("org.spockframework:spock-core:$spockVersion")
+    testImplementation("junit:junit:4.13.2")
```

Comment:
Could be replaced just by one dependency:
```
"org.spockframework:spock-junit4:$spockVersion"
```

Reply:
applied